### PR TITLE
Implement edit entry

### LINKED ISSUE #70: Refactor code so that schedule and tasks are under one class, Task

### PR DESCRIPTION
**Pull request by tsh22**

### CODE REVIEW

File: src/main/java/seedu/address/commons/core/Messages.java [hunk 1]
```diff
@@ -13,4 +13,5 @@
     public static final String MESSAGE_TASKS_LISTED_OVERVIEW = "%1$d tasks listed!";
     public static final String MESSAGE_TASK_DOES_NOT_EXIST = "The task provided does not exist";
     public static final String MESSAGE_SCHEDULE_DOES_NOT_EXIST = "The schedule provided does not exist!";
+    public static final String MESSAGE_ENTRY_DOES_NOT_EXIST = "The entry provided does not exist";
```
Comment (nicholastanvis) [inline src/main/java/seedu/address/commons/core/Messages.java:15]: keep this String, will resolve the conflict in #94 instead

File: src/main/java/seedu/address/logic/commands/EditEntryCommand.java [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+            + "Existing values will be overwritten by the input values.\n"
+            + "Parameters: ENTRY_NAME "
+            + "[" + PREFIX_NAME + "NAME] "
+            + "[" + PREFIX_START_DATE + "START_DATE] "
+            + "[" + PREFIX_END_DATE + "END_DATE] "
+            + "[" + PREFIX_TAG + "TAG]...\n"
+            + "Example: " + COMMAND_WORD + " Meeting "
+            + PREFIX_NAME + "Meeting with group ";
+
+    public static final String MESSAGE_EDIT_ENTRY_SUCCESS = "Edited entry: %1$s";
```
Comment (nicholastanvis) [inline src/main/java/seedu/address/logic/commands/EditEntryCommand.java]: i was thinking of moving constants to commons.core.Messages, going to do the same for #94

**Review by nicholastanvis — APPROVED** [review]

File: src/main/java/seedu/address/logic/commands/EditEntryCommand.java (hunk 2 above)
Comment (tsh22) [reply src/main/java/seedu/address/logic/commands/EditEntryCommand.java]: Sure sounds good

**Review by lrj689 — APPROVED** [review]

**Review by goatygoatygoat — APPROVED** [review]

**Comment by tsh22** [comment]
Checks not going through

**Pull request closed without merging**